Source/GSXib5KeyedUnarchiver.m

```diff
@@ -789,6 +799,10 @@ - (void) parser: (NSXMLParser*)parser
               // Need to store element for making the connections...
               [self addConnection: element];
             }
+          else if ([XmlConstraintRecordTags containsObject: elementName])
```

Comment:
As there is just one constraint element "constraint" you could test for that here directly. Or we could handle that as a connection. I am not sure whether the current code does anything useful with the constraint as you just decode it without storing it.